test-cases/gutenberg/writing-flow/undo-redo.md

```diff
@@ -26,18 +17,3 @@ Start on a new empty post.
 - Write some text on a text based block .
 - Press Undo until all new text has disappeared.
 - Press Redo to see the text appear again.
```

Comment:
@dcalhoun We are already covering the case of undo/redo text in Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/bb5c6d0ac1467f6c0cd771833b88586a53ed805a/test/native/integration/editor-history.native.js#L73-L95)), so I'm wondering if we could also remove this test case?

It's true that in the integration tests, we don't cover the known issues mentioned here, but maybe we could add new test cases to cover them. The only caveat I foresee is that we won't be able to check the cursor position. But definitely, that's something we could do in an E2E test.